### PR TITLE
Feat/react query pulse

### PR DESCRIPTION
**Pull request by t0rbik**
## PR-Codex overview
This PR removes several files and components, updates dependencies, and makes various code changes and additions, including:

- Adds an `EventEmitter` to the exports of a file
- Updates the `OPTION_TOKEN` constant value in `constants/contracts.ts`
- Changes text in `pages/claim/index.tsx`
- Updates the `QuoteSwapPayload` type in `pages/api/firebird-router.ts`
- Changes the import statement for `SwapComponent` in `pages/swap/index.tsx`
- Updates the `VestingInfoProps` interface in `components/ssVest/vestingInfo.tsx`
- Changes the image dimensions in `pages/configure/index.tsx`
- Changes the import statement for `LiquidityPairs` in `pages/liquidity/index.tsx`
- Updates the `MySnackbar` component in `components/snackbar/snackbar.tsx`
- Changes the `enhanceApp` function in `pages/_document.tsx`
- Adds `useAmounts` and `useIsWrapUnwrap` hooks in `components/liquidityManage/lib/useAmounts.ts` and `components/swap/lib/useIsWrapUnwrap.ts`, respectively
- Adds `SmallInput` component in `components/common/smallInput.tsx`
- Adds `useGauges` and `useBaseAssetWithInfoNoNative` hooks in `components/ssBribeCreate/queries.ts`
- Changes the import statements and adds a `CircularProgress` component in `components/liquidityPairs/LiquidityPairs.tsx`
- Updates the export statements in `constants/constants.ts`

### Detailed summary
- Removes several files and components
- Updates dependencies
- Adds and changes code and components in various files

> The following files were skipped due to too many changes: `Frontend-v1-Original/stores/constants/constants.ts`, `Frontend-v1-Original/components/navigation/navigation.tsx`, `Frontend-v1-Original/stores/helperStore.ts`, `Frontend-v1-Original/stores/types/types.ts`, `Frontend-v1-Original/utils/utils.ts`, `Frontend-v1-Original/components/swap/lib/usePriceDiff.ts`, `Frontend-v1-Original/pages/_app.tsx`, `Frontend-v1-Original/stores/constants/queryKeys.ts`, `Frontend-v1-Original/components/layout/layout.tsx`, `Frontend-v1-Original/components/ssVest/unlock.tsx`, `Frontend-v1-Original/components/ssLiquidityPairs/ssLiquidityPairsTable.tsx`, `Frontend-v1-Original/components/liquidityPairs/queries.ts`, `Frontend-v1-Original/components/ssVest/ssVest.tsx`, `Frontend-v1-Original/components/ssBribes/ssBribes.tsx`, `Frontend-v1-Original/components/ssBribes/mutations.ts`, `Frontend-v1-Original/components/ssVest/lockAmount.tsx`, `Frontend-v1-Original/components/ssVest/lockDuration.tsx`, `Frontend-v1-Original/components/ssVests/ssVests.tsx`, `Frontend-v1-Original/components/header/mobileHeader.tsx`, `Frontend-v1-Original/components/header/header.tsx`, `Frontend-v1-Original/components/transactionQueue/transaction.tsx`, `Frontend-v1-Original/components/snackbar/snackbarController.tsx`, `Frontend-v1-Original/components/ssVest/lock.tsx`, `Frontend-v1-Original/components/ssVotes/ssVotesTable.tsx`, `Frontend-v1-Original/components/common/massiveInput.tsx`, `Frontend-v1-Original/components/swap/routes.tsx`, `Frontend-v1-Original/components/ssVests/ssVestsTable.tsx`, `Frontend-v1-Original/components/ssRewards/ssRewardsTable.tsx`, `Frontend-v1-Original/components/ssVotes/lib/mutations.ts`, `Frontend-v1-Original/components/ssVotes/lib/queries.ts`, `Frontend-v1-Original/components/ssVest/MergeNFT.tsx`, `Frontend-v1-Original/components/ssVest/lib/queries.ts`, `Frontend-v1-Original/components/ssBribeCreate/mutations.ts`, `Frontend-v1-Original/components/header/info.tsx`, `Frontend-v1-Original/components/swap/lib/queries.ts`, `Frontend-v1-Original/lib/global/mutations.ts`, `Frontend-v1-Original/stores/constants/actions.ts`, `Frontend-v1-Original/components/ssBribeCreate/ssBribeCreate.tsx`, `Frontend-v1-Original/components/ssRewards/lib/queries.ts`, `Frontend-v1-Original/components/header/lib/queries.ts`, `Frontend-v1-Original/components/common/select.tsx`, `Frontend-v1-Original/components/swap/lib/mutations.ts`, `Frontend-v1-Original/components/ssRewards/ssRewards.tsx`, `Frontend-v1-Original/components/ssRewards/lib/mutations.ts`, `Frontend-v1-Original/components/transactionQueue/transactionQueue.tsx`, `Frontend-v1-Original/components/liquidityManage/LiquidityCreate.tsx`, `Frontend-v1-Original/components/liquidityManage/lib/queries.ts`, `Frontend-v1-Original/components/ssVotes/ssVotes.tsx`, `Frontend-v1-Original/components/swap/swap.tsx`, `Frontend-v1-Original/lib/global/queries.ts`, `Frontend-v1-Original/components/ssVest/lib/mutations.ts`, `Frontend-v1-Original/yarn.lock`, `Frontend-v1-Original/components/liquidityManage/lib/mutations.ts`, `Frontend-v1-Original/components/ssLiquidityManage/ssLiquidityManage.tsx`, `Frontend-v1-Original/lib/wagmiGen.ts`, `Frontend-v1-Original/legacy/stableSwapStore.ts`

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

**Comment by t0rbik** [comment]
It is basically the same but with pulse changes. Will test it as well

**Comment by t0rbik** [comment]
Tested manually, looks fine

File: Frontend-v1-Original/components/ssVests/ssVestsTable.tsx [hunk 1]
```diff
@@ -35,10 +35,11 @@ import relativeTime from "dayjs/plugin/relativeTime";
...
 
-import stores from "../../stores";
 import { formatCurrency } from "../../utils/utils";
-import { ACTIONS } from "../../stores/constants/constants";
 import { GovToken, VestNFT, VeToken } from "../../stores/types/types";
+import { useResetVest } from "../ssVest/lib/mutations";
+
+dayjs.extend(relativeTime);
 
 dayjs.extend(relativeTime);
```
Comment (oxSaturn) [inline Frontend-v1-Original/components/ssVests/ssVestsTable.tsx:44]:
```suggestion
dayjs.extend(relativeTime);

```

**Review by oxSaturn — APPROVED** [review]
Looks good after some manual tests of functions I used often. Hopefully we don't have such huge PRs in future as it's difficult to review.

**Pull request merged**